source/05-pages/page-wrappers/default.jsx

```diff
@@ -20,7 +20,12 @@ import { BackToTop } from '../../03-components/back-to-top/back-to-top.stories.j
 
 const PageWrapper = props => {
   // eslint-disable-next-line react/prop-types
-  const { children } = props;
+  const { children, front } = props;
```

Comment:
We’re also adding the `not-front` class in Drupal, so perhaps we should add it in Storybook as well?

Recommended:
```
  const { children, isHomepage } = props;

  if (isHomepage) {
    document.body.classList.add('front');
    document.body.classList.remove('not-front');
  } else {
    document.body.classList.add('not-front');
    document.body.classList.remove('front');
  }
```